src/Inventory.js

```diff
@@ -7,6 +7,7 @@ class Inventory extends Map {
   addItem(item) {
     this.set(item.uuid, item);
     item.isHeld = true;
+    item.container = this;
```

Comment:
I don't... like this. `container` is supposed to be an indicator of the _item_ it's currently contained in, not a reference to an `Inventory` object.